runtime/src/accounts_db.rs

```diff
@@ -5500,11 +5500,15 @@ impl AccountsDb {
                     drop(recycle_stores);
                     let old_id = ret.append_vec_id();
                     ret.recycle(slot, self.next_id());
+                    // This info show the appendvec history change history.  It helps debugging
+                    // the appendvec data corrupution issues related to recycling.
```

Comment:
I don't think a comment is necessary here (also fixes a typo by deleting it!)
```suggestion
```

Reply:
There was concern that this debug message was not related to this PR.  I put this comment here to explain why it should go with this PR.